2/Sum_bay.rb

```diff
@@ -0,0 +1,26 @@
+all_bay = Hash.new
+price_all = 0
+
+while true
+  puts "Введите название товара:"
+  product_name = gets.strip.upcase
+  break if product_name == "СТОП"
+
+  puts "Введите цену за 1 товара:"
+  product_price = gets.to_f
+
+  puts "Введите кол-во товара:"
+  product_count = gets.to_f
+
+  price = product_count * product_price
+  all_bay[product_name] = {"Цена за шт" => product_price, "Кол-во" => product_count, "Цена" => price}
+  price_all += price
+end
+
+puts "------\nИтого:\n------"
+
+all_bay.each do |key, value|
```

Comment:
```suggestion
cart.each do |product, info|
```
старайся давать переменным более понятные имена

Reply:
fixt